index.js

```diff
@@ -18,51 +19,24 @@ module.exports = {
       ],
     });
 
-    var chartistTreeSCSS = new Funnel(chartistPath, {
-      srcDir: 'scss',
-
-      include: [
-        '*.scss',
-      ],
-    });
-
-    return new MergeTrees([
+    return mergeTrees([
       vendorTree,
       chartistTree,
-      chartistTreeSCSS,
     ]);
   },
 
+  treeForStyles() {
```

Comment:
Oops yes. I can reimplement the old behavior of including CSS by default and letting the user opt into the Sass version. I was just tunnel-vision on getting the Sass import working.